src/components/Filter/Filter.tsx

```diff
@@ -0,0 +1,90 @@
+import { FC, useState } from 'react';
+import { NavLink } from 'react-router-dom';
+import cn from 'classnames';
+import { TodoList } from '../TodoList/TodoList';
+import { Todo } from '../../types/Todo';
+
+const FILTERS = {
```

Comment:
try to create `enum Filters` and use it everywhere

Reply:
@anstsot I think someone should change this lilne
const FILTERS = {
      all: ‘all’,
      completed: ‘completed’,
      active: ‘active’,
    };
in checklist.md
On first approach I made enum but I saw this and changed to const,
thanks for feedback ;-)
Happy Easter